packages/suite/src/actions/suite/metadataActions.ts

```diff
@@ -818,125 +807,487 @@ export const setDeviceMetadataKey =
                     },
                 },
             });
+
+            if (getState().metadata.failedMigration[device.state]) {
```

Comment:
I just tried current develop, enable local labelling, then loading this branch and got:
<img width="594" alt="image" src="https://github.com/trezor/trezor-suite/assets/3729633/eb72d3dc-8a1f-4cf1-8530-b7495b482158">
<img width="293" alt="image" src="https://github.com/trezor/trezor-suite/assets/3729633/c45d3e47-bdd3-45ea-a1e3-7fc4e6363778">

Reply:
and after reload of suite and enabling labeling again it works...

Reply:
but it is reproducible - happened to me on second try too

Reply:
fixed in 2bf788168c95b1ecd78d64d9db211160229f03f6